CONTRIBUTING.md

```diff
@@ -4,40 +4,39 @@ Contributions are always welcome, no matter how large or small. Before contribut
 please read the [code of conduct](CODE_OF_CONDUCT.md).
 
 ## Setup
-
 * Install Go 1.16
+* Install Docker 
 * Install [Soda CLI](https://gobuffalo.io/en/docs/db/toolbox)
-* Install Docker to run tests
-
-GoTrue uses the Go Modules support built into Go 1.11 to build. The easiest is to clone GoTrue in a directory outside of GOPATH, as in the following example:
-
-```sh
-$ git clone https://github.com/supabase/gotrue
-$ cd gotrue
-$ make deps
-```
-
-## Building
-
-```sh
-$ make build
-```
+  * `go install github.com/gobuffalo/pop/soda@latest`
+* Clone this repo: `git clone https://github.com/supabase/gotrue`
+* `cd gotrue`
+* To start the gotrue postgresql container running locally: `./hack/postgresd.sh` 
+* To compile the gotrue binary for execution: `make build` 
+* Before executing the binary (`./gotrue`), create a `.env` file in the root of the project and copy the following config in [example.env](example.env)
+* Gotrue requires a set of smtp credentials to run, you can generate your own smtp credentials via an smtp provider such as AWS SES, SendGrid, MailChimp, SendInBlue or any other smtp providers.
 
 ## Running database migrations for supabase
 - Create a `.env` file to store the custom gotrue environment variables. You can refer to an example of the `.env` file [here](hack/test.env)
-- Start PostgreSQL inside a docker container running `hack/postgresd.sh`
-- Build the gotrue binary `make build`
-- Execute the binary `./gotrue`
-  - gotrue runs any database migrations from `/migrations` on start 
+- Start PostgreSQL inside a docker container running `./hack/postgresd.sh`
+- Run `make migrate_test`
```

Comment:
Same in migration we need to check the script work from 0

Reply:
could you elaborate by what you mean by "work from 0" ?